---
float32 last_destination_x
float32 last_destination_y
